### ISSUE TITLE
Importing Netscape-formatted cookies ignores cookie with `#HttpOnly_` flag

### ISSUE THREAD

**Issue by noaione** [Moustachauve/cookie-editor#122]
Trying to import a cookies with a `#HttpOnly_` prefix will ignore that line since the code will skip anything starting with `#`: https://github.com/Moustachauve/cookie-editor/blob/v1.11.0/interface/popup/cookie-list.js#L302

As an example:
```
#HttpOnly_example.com	FALSE	/	FALSE	0	HttpOnlyTest	abc123
example.com	FALSE	/	FALSE	0	NoHttpOnlyTest	xyz987
```

Will only import `NoHttpOnlyTest` cookie but not `HttpOnlyTest` cookie.

![firefox_TtjiMG1S9Q](https://github.com/Moustachauve/cookie-editor/assets/34302902/39840474-b99c-4193-8b9d-0eb7d2a1c6a2)

**Comment by Moustachauve** [comment]
Yes, good find. Currently, all lines that start with # are ignored. In this case, "#HttpOnly_" seems to be a special flag that Cookie-Editor is not handling at all. 

I will add support for it in the next version as a priority. Adding support for HttpOnly for netscape format is pretty important.

**Comment by Moustachauve** [comment]
This is now fixed in version 1.12.0! This version is slowly rolling out for all browsers!
